tofhir-engine/src/test/scala/io/tofhir/test/engine/execution/FileStreamInputArchiverTest.scala

```diff
@@ -0,0 +1,129 @@
+package io.tofhir.test.engine.execution
+
+import io.tofhir.engine.config.ToFhirConfig
+import io.tofhir.engine.execution.{FileStreamInputArchiver, RunningJobRegistry}
+import io.tofhir.engine.model._
+import io.tofhir.engine.util.FileUtils
+import org.mockito.MockitoSugar._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.io.PrintWriter
+
+class FileStreamInputArchiverTest extends AnyFlatSpec with Matchers {
```

Comment:
Can you add tests for :
- delete mode
- multiple files
- validating the logic of getLastCommitOffset
- validating that only the recently committed files are archived but not the ones that are already archived